scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py

```diff
@@ -302,7 +303,8 @@ def update_copy_motr_config_file(self):
                    ("MOTR_M0D_DATA_DIR", f"{MOTR_M0D_DATA_DIR}"),
                    ("MOTR_M0D_CONF_XC", f"{MOTR_M0D_CONF_XC}"),
                    ("MOTR_M0D_ADDB_STOB_DIR", f"{MOTR_M0D_ADDB_STOB_DIR}"),
-                   ("MOTR_M0D_TRACE_DIR", f"{MOTR_M0D_TRACE_DIR}")]
+                   ("MOTR_M0D_TRACE_DIR", f"{MOTR_M0D_TRACE_DIR}"),
+                   ("MOTR_MOD_ADDB_RECORD_SIZE", f"{128M}")]
```

Comment:
Why is this required as a hardcoded parameter, can it be taken from some conf parameters instead

Reply:
If we dont want to harcode it then we can read it from solution.yaml as we are doing it with other config values.